docs/concepts/reftypes.mdx

```diff
@@ -0,0 +1,221 @@
+---
+title: Attached Artifacts
+sidebar_position: 1
+---
+
+# Attaching and listing reference types
+
+The OCI distribution specification allows us to "link" [OCI artifacts](artifact.mdx)
```

Comment:
Should we state the specific version of the OCI Distribution Spec? I prefer `
Since the [OCI Distribution Specification v1.1.0-rc.1](https://github.com/opencontainers/distribution-spec/releases/tag/v1.1.0-rc1), it allows us to "link" [OCI artifacts](artifact.mdx)`

Reply:
I'll add the details to the latter part in a subsequent section. Pointing to a point in time concept and liking to the spec in the first sentence seems too dense.

Reply:
Also the concept of distribution is not needed to create refereces since its can be on the client side also without using references.